e2e/cypress/integration/menus/status_dropdown_spec.js

```diff
@@ -181,4 +182,33 @@ describe('Status dropdown menu', () => {
                 and('have.text', 'Status').and('not.have.css', 'cursor', 'pointer');
         });
     });
+
+    describe('Custom status option in Status dropdown menu', () => {
+        before(() => {
+            cy.apiUpdateConfig({TeamSettings: {EnableCustomUserStatuses: true}});
+            cy.apiClearUserCustomStatus();
+        });
+
+        it('should show Set a Custom Status header when EnableCustomStatuses option is enabled, with cursor pointer', () => {
+            // # Wait for posts to load
+            cy.get('#postListContent').should('be.visible');
+
+            // # Open status menu
+            cy.get('.MenuWrapper .status-wrapper.status-selector button.status').click();
+
+            // # Verify "Set a Custom Status" header is visible
+            cy.get('.MenuWrapper.status-dropdown-menu .Menu__content.dropdown-menu li#status-menu-custom-status').should('be.visible').
+                and('have.text', 'Set a Custom Status').and('have.css', 'cursor', 'pointer');
+        });
+
+        specify('clear button should not be visible if status is not set', () => {
+            // # Wait for posts to load
+            cy.get('#postListContent').should('be.visible');
+
+            // # Open status menu
+            cy.get('.MenuWrapper .status-wrapper.status-selector button.status').click();
+
+            cy.get('.MenuWrapper.status-dropdown-menu .Menu__content.dropdown-menu li#status-menu-custom-status #custom_status__clear').should('not.exist');
```

Comment:
Could you add a comment to this one?

Also, do we need to be this specific with the selector? Would something like `.status-dropdown-menu #custom_status__clear` work here instead, or do you think it's better to be more specific here? I could go either way with that.